kmip/pie/client.py

```diff
@@ -541,6 +542,64 @@ def activate(self, uid=None):
             message = result.result_message.value
             raise exceptions.KmipOperationFailure(status, reason, message)
 
+    def revoke(self, revocation_reason, uid=None, revocation_message=None,
+               compromise_occurrence_date=None):
+        """
+        Revoke a managed object stored by a KMIP appliance.
+
+        Args:
+            revocation_reason (RevocationReasonCode): An enumeration indicating
+                the revocation reason.
+            uid (string): The unique ID of the managed object to revoke.
```

Comment:
OK. I also added that for the activate operation in this commit. Let me know if you want a separate commit.